scene/gui/color_picker.cpp

```diff
@@ -1993,14 +2168,36 @@ ColorPicker::ColorPicker() {
 
 	preset_group.instantiate();
 
+	HBoxContainer *palette_box = memnew(HBoxContainer);
+	palette_box->set_h_size_flags(SIZE_EXPAND_FILL);
+	real_vbox->add_child(palette_box);
+
 	btn_preset = memnew(Button);
 	btn_preset->set_text("Swatches");
 	btn_preset->set_flat(true);
 	btn_preset->set_toggle_mode(true);
 	btn_preset->set_focus_mode(FOCUS_NONE);
 	btn_preset->set_text_alignment(HORIZONTAL_ALIGNMENT_LEFT);
 	btn_preset->connect("toggled", callable_mp(this, &ColorPicker::_show_hide_preset).bind(btn_preset, preset_container));
-	real_vbox->add_child(btn_preset);
+	palette_box->add_child(btn_preset);
+
+	HBoxContainer *padding_box = memnew(HBoxContainer);
+	padding_box->set_h_size_flags(SIZE_EXPAND_FILL);
+	palette_box->add_child(padding_box);
+
+	menu_btn = memnew(Button);
+	menu_btn->set_flat(true);
+	menu_btn->set_tooltip_text(ETR("Show all options available."));
+	menu_btn->set_focus_mode(FOCUS_NONE);
+	menu_btn->connect(SceneStringName(pressed), callable_mp(this, &ColorPicker::_update_menu));
+	palette_box->add_child(menu_btn);
+	options_menu = memnew(PopupMenu);
```

Comment:
And same here. See LineEdit for example of optional Controls:
https://github.com/godotengine/godot/blob/daa81bbb7d1c6d75d1711595604178ee62a5801d/scene/gui/line_edit.cpp#L2427

Reply:
Make sure to use `force_parent_owned()` for the time being, or it'll be duplicated when the picker is (there's work for resolving that as well generally and specifically)